ETL/DTO/EmpDataRow.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Globalization;
+
+
+namespace ETL.DTO
+{
+  public class EmpDataRow
```

Comment:
does Emp stand for Empty?  Why abbreviate?

Reply:
I'm having Emp stand for Employee.  Otherwise Employee is duplicated everywhere and names like "EmployeeDataParser" get really long.  I'm open to changing it.